### PR TITLE
Make pilot-agent 15mb smaller

### PR DESCRIPTION
**Pull request by howardjohn**
Drop a constant import to avoid importing all of pilot

### CODE REVIEW

**Comment by howardjohn** [comment]
found by https://github.com/loov/goda for what its worth

File: pkg/istio-agent/sds-agent.go [hunk 1]
```diff
@@ -120,6 +119,8 @@ const (
 	InitialBackoff = "INITIAL_BACKOFF_MSEC"
 
 	pkcs8Key = "PKCS8_KEY"
+
+	CACertNamespaceConfigMapDataName = "ca-cert-ns.pem"
```
Comment (nmittler) [inline pkg/istio-agent/sds-agent.go]: Can we move this to `/pkg/config/constants`?

Comment (howardjohn) [reply pkg/istio-agent/sds-agent.go]: done

**Comment by kyessenov** [comment]
Lol, this is why having a package of constants is not a good idea :) If it's truly a constant, it should be safe to copy by value.

**Pull request merged**